gcloud-java-logging/src/test/java/com/google/cloud/logging/LoggingLevelTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.logging;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import java.util.logging.Level;
+
+public class LoggingLevelTest {
+
+  @Test
+  public void testDebug() {
+    LoggingLevel debug = LoggingLevel.DEBUG;
+    assertEquals(Severity.DEBUG, debug.severity());
+    assertEquals("DEBUG", debug.getName());
+    assertTrue(debug.intValue() < Level.FINEST.intValue());
+    assertTrue(debug.intValue() > Level.ALL.intValue());
+  }
+
+  @Test
```

Comment:
Yep - figured that out.  I figured it would look like a rant when I wrote it, and after I spent a bit more time, I realized why you were doing what you were doing.